lib/sufia.rb

```diff
@@ -20,6 +20,8 @@
 require 'sufia/inflections'
 require 'kaminari_route_prefix'
 
+require 'sufia/hydra_derivatives_compat'
+
```

Comment:
I think once https://github.com/samvera/hydra-derivatives/pull/182 is merged and 3.4.1 is released on rubygems, you can swap this with a `bundle update hydra-derivatives`